.changelog/2420.txt

```diff
@@ -0,0 +1,64 @@
+```release-note:note
```

Comment:
Alternatively can consider combining entries like below:

```release-note:note
resource/mongodbatlas_advanced_cluster: Deprecates following attributes - `replication_specs.#.id`, `replication_specs.#.num_shards`, `disk_size_gb`, `advanced_configuration.0.default_read_concern`, and  `advanced_configuration.0.fail_index_key_too_long`.
```

Reply:
not sure, probably I'd use this with all attributes together